src/dotnet/commands/dotnet-vstest/VSTestForwardingApp.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.Cli
+{
+    public class VSTestForwardingApp
+    {
+        private const string vstestExename = "vstest.console.dll";
```

Comment:
changed vstestExename to VstestAppName. 
Yes this will work for full framework as well. In this case also vstest.console.dll will get call from dotnet and vstest.console.dll will call testhost.exe instead of testhost.dll.